workspaces/github-actions/plugins/github-actions/README.md

```diff
@@ -64,6 +80,36 @@ const serviceEntityPage = (
 3. Run the app with `yarn start` and the backend with `yarn start-backend`.
    Then navigate to `/github-actions/` under any entity.
 
+### Integrating with `EntityPage` (New Frontend System)
+Follow this section if you are using Backstage's [new frontend system](https://backstage.io/docs/frontend-system/).
+
+1. Import `githubActionsPlugin` in your `App.tsx` and add it to your app's `features` array:
```

Comment:
from what I understand it's the App.tsx in the app-next folder. maybe add it in the readme?

Reply:
`app-next` is a convention when having multiple apps in a repo using different versions. Adopters typically wouldn't do that, but instead have a single `app` package that picks one version of the frontend system.